install_config/install/prerequisites.adoc

```diff
@@ -140,6 +142,9 @@ a master host in an {product-title} cluster of 2000 pods would be the minimum
 requirements of 2 CPU cores and 16 GB of RAM, plus 2 CPU cores and 3 GB of RAM,
 totaling 4 CPU cores and 19 GB of RAM.
 
+If etcd is co-located, there is just no way 2 cores will work, and probably 4 will
+be too little for even a moderately sized/loaded cluster.
+
```

Comment:
Just a nitpick, but this statement and the similar bullet in Minimum Requirements seem redundant.    Suggest removing or maybe rewording it to "Masters with a co-located etcd require a minimum of 4 cores.  2 core systems will not work."